reference/snapshot.md

```diff
@@ -17,15 +16,16 @@ Use cases for volume snapshots include:
 Volume snapshots allow the creation of read-only incremental copies of volumes, enabling you to maintain a history of your data. These volume snapshots possess the following characteristics:
 - **Consistency**: The data stored within a snapshot remains consistent across all replicas of the volume, whether local or remote.
 - **Immutability**: Once a snapshot is successfully created, the data contained within it cannot be modified.
-- **Thin Provisioning**: Thin provisioning will be enabled explicitly when creating the volume. It allows volumes to be allocated with minimal initial capacity and dynamically grow as data is written.
-
 
 Currently, Mayastor supports the following operations related to volume snapshots:
 
 1. Creating a snapshot for a PVC
 2. Listing available snapshots for a PVC
 3. Deleting a snapshot for a PVC
 
+:::note
+This version of Mayastor does not support creation of snapshot clones or creation volumes out of snapshots.
```

Comment:
Grammatical typo in "or creation volumes out of snapshots"?
Maybe re-phrase as "The current version does not support creation of a clone or a new volume from a snapshot."